src/test/java/com/google/neighborgood/IntegrationTest.java

```diff
@@ -310,36 +292,23 @@ public void _07_Userpage_AsLoggedUser_VerifyCompletedTask() {
     // Location of first task listed in the await verification table
     String awaitVerifTaskXPath = "//tbody[@id='await-verif-body']/tr[1]";
 
-    String taskOverview =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(awaitVerifTaskXPath + "/td[1]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("overview"), taskOverview);
-
-    String taskHelper =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(awaitVerifTaskXPath + "/td[2]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("helper"), taskHelper);
-
-    String taskStatus =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(awaitVerifTaskXPath + "/td[3]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("status"), taskStatus);
-
+    // Content targets that will be verified
+    String[] taskContentsTarget = {"overview", "helper", "status"};
+
+    // Iterates over task elements and compares the element's text with that of the stored
+    // recentTask
+    for (int i = 0; i < taskContentsTarget.length; i++) {
+      String taskContentItem =
+          wait.until(
+                  new Function<WebDriver, WebElement>() {
+                    public WebElement apply(WebDriver driver) {
+                      return driver.findElement(By.xpath(awaitVerifTaskXPath + "/td[1]"));
```

Comment:
Should this line be `awaitVerifTaskXPath + "/td[" + Integer.toString((i + 1)) + "]"))`?

Reply:
Ah I saw you have fixed this in one commit.